### PR TITLE
[MLIR][TORCH] Add folder for torch_c.from_i64 & torch_c.to_i64

### PR DESCRIPTION
**Pull request by fortianyou**

### CODE REVIEW

File: test/Dialect/TorchConversion/canonicalize.mlir [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// RUN: torch-mlir-opt %s -canonicalize | FileCheck %s
+
+// CHECK-LABEL:  func.func @torch_c.from_i64_to_i64() -> !torch.vtensor<[4],si64> {
+// CHECK:            %[[CST:.*]] = arith.constant dense<[5, 1, 4, 2]> : tensor<4xi64>
+// CHECK:            %[[T0:.*]] = torch_c.from_builtin_tensor %cst : tensor<4xi64> -> !torch.vtensor<[4],si64>
+// CHECK:            return %[[T0:.*]] !torch.vtensor<[4],si64>
+func.func @torch_c.from_i64_to_i64() -> !torch.vtensor<[4],si64> {
```
Comment (silvasean) [inline test/Dialect/TorchConversion/canonicalize.mlir:7]: Can you add unit tests for each individual fold? Having one big function like this makes it hard to debug failures later.

See this file for examples: https://github.com/llvm/torch-mlir/blob/main/test/Dialect/Torch/canonicalize.mlir

**Review by silvasean — CHANGES_REQUESTED** [review]

File: test/Dialect/TorchConversion/canonicalize.mlir (hunk 1 above)
Comment (fortianyou) [reply test/Dialect/TorchConversion/canonicalize.mlir]: done

**Comment by fortianyou** [comment]
@silvasean I have split the ut into tiny cases. Do you have any other comments?

**Comment by silvasean** [comment]
Looks good!

**Review by silvasean — APPROVED** [review]

**Comment by silvasean** [comment]
Sorry for the delay on this -- I was on vacation.

**Pull request merged**